assets/prometheus-k8s/rules.yaml

```diff
@@ -291,6 +300,12 @@ spec:
       record: cluster:node_instance_type_count:sum
     - expr: sum(etcd_object_counts) BY (instance)
       record: instance:etcd_object_counts:sum
+    - expr: sum((cluster:master_nodes * on(node) group_left kube_node_status_capacity_cpu_cores)
+        or on(node) (cluster:infra_nodes * on(node) group_left kube_node_status_capacity_cpu_cores)
```

Comment:
So this won't tell me that it's both a master and infra node?

Reply:
So if a node is marked `master` and `infra` it would be bucket into `master` because BY condition and its order.